frontend/src/plugins/guide/listener.ts

```diff
@@ -49,8 +61,9 @@ const initLocationListener = () => {
 
 // initStorageListener detecting storage changes
 const initStorageListener = () => {
-  window.addEventListener("storage", () => {
+  window.addEventListener("storage", async () => {
     console.log("storage changed");
```

Comment:
Console.log should be removed in the future.

Reply:
WIP